ui/src/app/routing/app-routing.tsx

```diff
@@ -329,9 +329,7 @@ export const AppRoutingComponent: React.FunctionComponent<
                 fontFamily: 'Montserrat',
               }}
             >
-              <div>
-                <img alt='logo' src={logo} width='155px' />
-              </div>
+              <SignedInAouHeaderWithDisplayTag />
```

Comment:
I can't confirm that this looks the same, because of https://precisionmedicineinitiative.atlassian.net/browse/RW-9020